proto/viam/app/data/v1/data.proto

```diff
@@ -55,6 +55,10 @@ service DataService {
   // BoundingBoxLabelsByFilter gets all string labels for bounding boxes from data based on given filter.
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
 
+  // ConfigureDatabaseUser configures a database user for the Viam organization. It can also be used to
+  // reset the password of an existing database user.
```

Comment:
Can we add some context to the comment to distinguish this from other mongo dbs? Can be something as simple as "... configures a _federated_ database user ...."

Reply:
Yes good point, this isn't very clear. Added more to the comment!